scss/common/_dragclue.scss

```diff
@@ -1,23 +0,0 @@
-@include exports( "common/dragclue" ) {
-
-    // Layout
-    .k-drag-clue {
```

Comment:
These styles seem to be still used in the TreeList, TreeView, and Grid:

![image](https://user-images.githubusercontent.com/90405/29808978-f53dbf4a-8ca2-11e7-97a0-24f97b22f596.png)

Reply:
the styles are not removed. just singled out. They already exist in base: https://github.com/telerik/kendo-theme-default/blob/647f69552b6ebe8d4aecb6a484e8615ce19f11bc/scss/common/_base.scss#L596-L618